kale/loaddata/multi_domain.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         target_sampling_config=None,
         n_fewshot=None,
         random_state=None,
+        sub_class_ids=None,
```

Comment:
See about for all. sub_class could be misleading. Subclasses are finer classes of a class.